lib/model/QubitDigitalObject.php

```diff
@@ -2617,47 +2617,24 @@ public static function convertAudioToMp3($originalPath, $newPath)
     public function createVideoDerivative($usageId, $connection = null)
     {
         // Build new filename and path
-        $originalFullPath = $this->getAbsolutePath();
-        list($originalNameNoExtension) = explode('.', $this->getName());
```

Comment:
Will this work for files with weird filenames where there's a period in a name?

Like "bob.comedy.mp4".

Seems like this might not matter though as the filename's going to be an internal filename anyways.

Reply:
Good point!  This logic does cause the derivative filenames to be truncated (Your eg would become bob_141.mp4).  This same logic is used for audio files and images currently as well. It should be pretty easy to update to use PHP's pathinfo() to get the filename in each circumstance.